release/rllib_tests/learning_tests/yaml_files/ddpg/ddpg-hopperbulletenv-v0.yaml

```diff
@@ -10,7 +10,7 @@ ddpg-hopperbulletenv-v0:
     config:
         actor_hiddens: [256, 256]
         critic_hiddens: [256, 256]
-        n_step: 3
+        n_step: 1
```

Comment:
Here, too, remove rollout_fragment_length (or set to "auto") further below entirely.